### PR TITLE
Release v1.3.0

### PR DESCRIPTION
**Pull request by Ivan-Velickovic**
Depends on https://github.com/seL4/microkit/pull/152.

### CODE REVIEW

**Review by lsf37 — APPROVED** [review]

File: CHANGES.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # Revision History for Microkit
 
+## Release 1.3.0-dev
+
```
Comment (lsf37) [inline CHANGES.md:4]: This part should probably be in a separate commit, because this is the commit we're going to tag as release (so we don't want the upcoming `-dev`  yet)

Comment (Ivan-Velickovic) [reply CHANGES.md:4]: Ah okay will split it up into a separate commit.

Comment (Ivan-Velickovic) [reply CHANGES.md:4]: Done, made the change to `VERSION.md` as well.
